WooCommerce/WooCommerceTests/ViewRelated/Dashboard/DashboardViewModelTests.swift

```diff
@@ -609,6 +609,143 @@ final class DashboardViewModelTests: XCTestCase {
         // Then
         XCTAssertNil(analyticsProvider.receivedEvents.firstIndex(of: "dashboard_store_timezone_differ_from_device"))
     }
+
+    // MARK: Blaze banner
+    // swiftlint:disable:next line_length
+    func test_updateBlazeBannerVisibility_updates_showBlazeBanner_to_true_if_site_is_eligible_for_blaze_and_banner_is_not_dismissed_yet_and_store_has_products() async throws {
+        // Given
+        let checker = MockBlazeEligibilityChecker(isSiteEligible: true)
+        let uuid = UUID().uuidString
+        let userDefaults = try XCTUnwrap(UserDefaults(suiteName: uuid))
+        let viewModel = DashboardViewModel(siteID: sampleSiteID,
+                                           stores: stores,
+                                           userDefaults: userDefaults,
+                                           blazeEligibilityChecker: checker)
+        XCTAssertFalse(viewModel.showBlazeBanner)
+
+        // When
+        stores.whenReceivingAction(ofType: ProductAction.self) { action in
+            switch action {
+            case .checkIfStoreHasProducts(_, let onCompletion):
+                onCompletion(.success(true))
+            default:
+                break
+            }
+        }
+        await viewModel.updateBlazeBannerVisibility()
+
+        //  Then
+        XCTAssertTrue(viewModel.showBlazeBanner)
+    }
+    // swiftlint:enable:next line_length
```

Comment:
We don't have to enable it again as you are using `swiftlint:disable:next` to disable it only for the next line.

Reply:
TIL! Removed in 8bab03bc8b9240ba7bcd729c9a20fd9881fe0fcc.